code/modules/power/apc.dm

```diff
@@ -128,8 +136,24 @@
 		src.update_icon()
 		spawn(5)
 			src.update()
-
-
+			
+/obj/machinery/power/apc/Destroy()
+	if(malfai && operating)
+		if (ticker.mode.config_tag == "malfunction")
+			if (src.z == 1) //if (is_type_in_list(get_area(src), the_station_areas))
+				ticker.mode:apcs--
+	area.power_light = 0
+	area.power_equip = 0
+	area.power_environ = 0
+	area.power_change()
+	if(occupier)
+		malfvacate(1)
+	qdel(wires)
+	if(cell)
+		qdel(cell) // qdel
+	if(terminal)
+		disconnect_terminal()
+	..()

```

Comment:
Destroy() procs should always return the parent value, either by `return ..()` or `. = ..()`